dm/master/config.go

```diff
@@ -32,6 +33,8 @@ import (
 // and assign it to SampleConfigFile while we build dm-master
 var SampleConfigFile string
 
+var defaultRPCTimeout = "30s"
```

Comment:
why does it have different value with `defaultRPCTimeout` in _dmctl_?

Reply:
in the current design, each RPC from dm-master to dm-worker should return as soon as possible, so I use a short default value here. While dm-master may distribute RPC calls to a large scale of dm-workers, so the default time for dmctl is set to a large number.